### ISSUE TITLE
Update frequency code?

### ISSUE THREAD

**Issue by MatthewCroughan** [fellinga/node-red-contrib-ui-time-scheduler#2]
Hey! This massively simplifies my life, but it'd be fantastic if we could adjust the frequency of emissions of true/false. Where is the code that defines this?

**Comment by fellinga** [comment]
I am glad it simplifies things for you. So you want to change the update interval (60 seconds currently) to any other number, is this correct?

**Comment by MatthewCroughan** [comment]
That's right. This should be exposed in the configuration of the node, preferably since it will be a constant from the perspective of the UI user.

**Comment by fellinga** [comment]
Okay I see. I have actually been thinking about this for a long time now, so I guess it is time to do something about it. I am going to implement this tonight and update this issue when I am done. Thank you for your feedback.

**Comment by MatthewCroughan** [comment]
If you're interested in my use case, it's going to be for timing a massive light show similar to [this project](https://twitter.com/HullMaritime/status/1222585210719625218?s=19) I worked on, which uses nodered on a raspberry pi to send OSC/Artnet signals to trigger events on a given schedule. :smile: 

My eventual blog post will arrive with this project given a massive mention, since it basically does everything I tried to do purely with nodes nicely, and exposes it as a nice user interface. This is similar, but fails to provide the input sanitizing interface that you have.

https://discourse.nodered.org/t/two-subflows-i-made-a-timerange-switch-and-a-scheduler-dashboard-integratable/24586/2
